nilearn/decoding/proximal_operators.py

```diff
@@ -248,8 +244,7 @@ def _prox_tvl1(
                 )
                 if verbose:
                     print(
-                        "\tProxTVl1: Iteration % 2i, dual gap: % 6.3e"
-                        % (i, dgap)
+                        "\tProxTVl1: Iteration {i: 2}, dual gap: {dgap: 6.3e}"
```

Comment:
isn't this supposed to be an f-string ?

Reply:
oh gosh yes!!!
thanks for catching this

Reply:
fixed